src/Task/TwigCs.php

```diff
@@ -57,6 +59,12 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->addOptionalArgument('--ruleset=%s', $config['ruleset']);
         $arguments->addOptionalArgument('--ansi', true);
 
+        if (\is_array($config['exclude'])) {
```

Comment:
This will always be an array since it is enforced by the options resolver

Reply:
Yes, of course :smile: